### PR TITLE
update comments on base classes to reflect current standards

### PR DESCRIPTION
**Pull request by m-nash**

### CODE REVIEW

File: Azure.ResourceManager.Core/ResourceOperationsBase.cs [hunk 1]
```diff
@@ -71,18 +71,16 @@ protected ResourceOperationsBase(AzureResourceManagerClientOptions options, Reso
         }
 
         /// <summary>
-        /// Get details for this resource from the service.  This call will block until a response is returne from the service
+        /// Get details for this resource from the service.
```
Comment (YalinLi0312) [inline Azure.ResourceManager.Core/ResourceOperationsBase.cs:74]: Gets

File: Azure.ResourceManager.Core/ResourceOperationsBase.cs [hunk 2]
```diff
@@ -71,18 +71,16 @@ protected ResourceOperationsBase(AzureResourceManagerClientOptions options, Reso
...
         /// </summary>
-        /// <returns> An Http response with the operations for this resource. </returns>
+        /// <returns> A response with the <see cref="ArmResponse{TOperations}"/> operation for this resource. </returns>
         public abstract ArmResponse<TOperations> Get();
 
         /// <summary>
-        /// Get details for this resource from the service.  This call returns a Task, which can  be used to control waiting
-        /// for a response from the service.
+        /// Get details for this resource from the service.
```
Comment (YalinLi0312) [inline Azure.ResourceManager.Core/ResourceOperationsBase.cs:80]: Gets

**Review by YalinLi0312 — CHANGES_REQUESTED** [review]
Add two minor comments. And I think the class summary in ResourceOperationsBase class should follow our latest standard.

**Review by allenjzhang — APPROVED** [review]
With Yalin's comments.

**Review by YalinLi0312 — APPROVED** [review]

**Pull request merged**